src/types/api-responses.ts

```diff
@@ -24,6 +25,26 @@ export interface ICreateGroupResponse {
   message: string
 }
 
+export interface IFetchDatasets {
```

Comment:
Hm, I usually go towards using `Array<Type>` instead of `Types` and `Type`. How do you feel?

Reply:
`Array<Type>` sounds good to me.